test/src/source-map-cache.js

```diff
@@ -35,10 +35,16 @@ try {
 
 var sourceMapCache = new SourceMapCache()
 _.forOwn(covered, function (fixture) {
-  sourceMapCache.add(fixture.relpath, fixture.contentSync())
+  var source = fixture.contentSync()
+  var sourceMap = convertSourceMap.fromSource(source) || convertSourceMap.fromMapFileSource(source, fixture.relpath)
+  if (sourceMap) {
+    sourceMapCache.addMap(fixture.relpath, sourceMap.sourcemap)
+  }
 })
 
-var coverage = ap(require('../fixtures/coverage'))
+var getReport = function () {
+  return _.cloneDeep(require('../fixtures/report'))
```

Comment:
Looks like another place I screwed up the rebase.

> posix-style paths

yeah, might not even make sense to request that change. Since the spec demands absolute paths anyways, that would make windows paths look really weird: `C:/foo/bar`